### PR TITLE
added operations module

### PR DESCRIPTION
**Pull request by Harshvardhan80**
Targets #2783 

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Operations/Resources/views/layouts/master.blade.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+       {{-- <link rel="stylesheet" href="{{ mix('css/operations.css') }}"> --}}
+
+    </head>
+    <body>
+        @yield('content')
+
+        {{-- Laravel Mix - JS File --}}
+        {{-- <script src="{{ mix('js/operations.js') }}"></script> --}}
+    </body>
+</html>
```
Comment (GauravGusain98) [inline Modules/Operations/Resources/views/layouts/master.blade.php:19]: This is not correct way to create a page. please check logic in other modules

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Along with the feedbacks, also Remove package-lock.json file changes.

File: Modules/Operations/Resources/views/layouts/master.blade.php (hunk 1 above)
Comment (Harshvardhan80) [reply Modules/Operations/Resources/views/layouts/master.blade.php:19]: ok sir

File: resources/views/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@extends('layouts.app')
```
Comment (GauravGusain98) [inline resources/views/index.blade.php:1]: The blade file corresponding to Operations should be inside operation module

Comment (GauravGusain98) [reply resources/views/index.blade.php:1]: This file is not needed

File: resources/views/layouts/navbar.blade.php [hunk 3]
```diff
@@ -113,14 +113,28 @@
             </div>
         </li>
     @endcan
+    @can('operations.view')
+        <li class="nav-item dropdown">
+            <a id="navbarDropdown_operations" class="nav-link dropdown-toggle" href="#" role="button"
+                data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" v-pre>Operations <span class="caret"></span>
+            </a>
+            <div class="dropdown-menu z-index-1100" aria-labelledby="navbarDropdown_operations">
+                <a class="dropdown-item" href="{{ url('/operations') }}">Office location</a>
```
Comment (GauravGusain98) [inline resources/views/layouts/navbar.blade.php:122]: We use route for URLs. PLease use route function

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**